corehq/apps/app_manager/models.py

```diff
@@ -1519,18 +1520,48 @@ class MappingItem(DocumentSchema):
     value = DictProperty()
 
     @property
+    def treat_as_expression(self):
+        """
+        Returns if whether the key can be treated as a valid expression that can be included in
+        condition-predicate of an if-clause for e.g. if(<expression>, value, ...)
+        """
+        special_chars = '{}()[]=<>."\'/'
```

Comment:
My thought was that it's an effort to think, "What are all of the specific characters that might indicate an expression?" and easier to think, "An expression is anything more complex than letters, numbers, and spaces."

...if that still doesn't make sense, don't worry about it, not a big deal.